source/getEntityTypescriptPostgres.ts

```diff
@@ -1,169 +1,522 @@
-import {createPostgresDataProvider} from 'remult/postgres';
-import {writeFile, mkdir} from 'fs/promises';
+import {
+	existsSync,
+	lstatSync,
+	mkdirSync,
+	readdirSync,
+	rmdirSync,
+	unlinkSync,
+	writeFileSync,
+} from 'fs';
+import {green, yellow} from 'kleur/colors';
+import {SqlDatabase, remult} from 'remult';
 
-export function generateModels(
-	connectionString: string,
-	tables: string[],
-	dir = '.',
+const deleteFolderRecursive = path => {
```

Comment:
i think you should add a warning that we will override

Reply:
Yes, would be good to have a flag to write everything or not. Should we add some tasks? or?